examples/ephemeral/index.html

```diff
@@ -10,10 +10,10 @@
 
   <script>
     async function onLoaded(client) {
-      thread_id = await client.threadMake("@support");
-      await client.threadOpen(thread_id);
+      threadId = await client.threadMake("@support");
```

Comment:
no var declaration here.  I recommend eslint for catching this stuff.  Might have to snip your js out to a seperate file to run eslint on in it, but it's  a nice check to run on code we expect people to look at.  Also, grab the .eslint.json file from the root of the project for ref on guidelines we (I've) typically used.